tools/docker/docker-compose.yml

```
@@ -12,6 +12,8 @@ services:
         "-c",
         "alembic upgrade head && ansible-events-ui"
       ]
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
```

This is needed for RHEL or Fedora hosts with SElinux enabled: 
```suggestion
      - "/var/run/docker.sock:/var/run/docker.sock:z
```